projects/lxmert/configs/coco/masked.yaml

```diff
@@ -0,0 +1,58 @@
+dataset_config:
+  masked_coco:
+    data_dir: ${env.data_dir}/datasets
+    depth_first: false
+    fast_read: false
+    use_images: false
+    use_features: true
+    use_image_feature_masks: true
+    return_features_info: true
+    max_features: 36
+    use_ocr: false
+    use_ocr_info: false
+    two_sentence: false
+    two_sentence_probability: 0
+    false_caption: false
+    false_caption_probability: 0.5
+    features:
+      train:
+      - coco/defaults/features/trainval2014.lmdb
+      - coco/defaults/features/trainval2014.lmdb
+    annotations:
+      train:
+      - coco/defaults/annotations/imdb_karpathy_train_by_image.npy
+      - coco/defaults/annotations/imdb_karpathy_val_by_image.npy
+    processors:
+      bbox_processor:
+        type: bbox
+        params:
+          max_length: 36
+      text_processor:
+        type: bert_tokenizer
+        params:
+          tokenizer_config:
+            type: bert-base-uncased
+            params:
+              do_lower_case: true
+          max_seq_length: 20
+      attribute_processor:
+        type: vocab
+        params:
+          max_length: 1
+          vocab:
+            type: random
+            vocab_file: vocabs/vocabulary_100k.txt
+      masked_token_processor:
+        type: masked_token
+        params:
+          tokenizer_config:
+            type: bert-base-uncased
+            params:
+              do_lower_case: true
+          mask_probability: 0.15
+          max_seq_length: 20
```

Comment:
Increase this to a larger value if some captions are longer than 20 tokens.

Reply:
Thanks for pointing this out. Original Implementation of LXMERT uses 20. So, we might want to keep it 20.